app/controllers/idv/session_errors_controller.rb

```diff
@@ -51,5 +55,13 @@ def confirm_idv_session_step_needed
       return unless user_fully_authenticated?
       redirect_to idv_phone_url if idv_session.profile_confirmation == true
     end
+
+    def set_try_again_path
+      if request.referer&.starts_with? idv_in_person_url
```

Comment:
I don't have a lot of confidence in the `referer` attribute, since it's easy for it to get lost (e.g. does a reload retain referer? Not really sure tbh). I wonder if we could include a parameter in the URL when redirecting to these error pages in [`DocAuthBaseStep#idv_failure`](https://github.com/18F/identity-idp/blob/1986d428e1f644f278a94d70f7061e728ecb3760/app/services/idv/steps/doc_auth_base_step.rb#L24-L36), then use that when determining the `try_again_path`.

Or, alternatively, we may be able to infer that the user is in the in-person flow if we `user_session['idv/in_person']` is not empty.

Reply:
This seemed like it would work in most practical use cases & kept the referrer when I refreshed. I understand the concern, though, and refactored it accordingly.